integrasjonspunkt/src/main/java/no/difi/meldingsutveksling/IntegrasjonspunktApplication.java

```diff
@@ -45,15 +50,28 @@ public static void main(String[] args) {
                 return;
             }
 
-            new SpringApplicationBuilder(IntegrasjonspunktApplication.class)
+            ConfigurableApplicationContext context = new SpringApplicationBuilder(IntegrasjonspunktApplication.class)
                     .initializers(new SpringCloudProtocolResolver())
                     .run(args);
 
+//            IntegrasjonspunktProperties.NextMove.ServiceBus serviceBusProps = context.getBean(IntegrasjonspunktProperties.class).getNextmove().getServiceBus();
+//            if (serviceBusProps.isBatchRead()) {
+//                pollServiceBus(context);
+//            }
+
         } catch (SecurityException se) {
             logMissingJCE(se);
         }
     }
 
+//    private static void pollServiceBus(ApplicationContext context) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3ACommentedOutCodeLine)